aea/agent.py

```diff
@@ -112,15 +97,25 @@ def _get_runtime_class(self) -> Type[BaseRuntime]:
             )
         return self.RUNTIMES[self._runtime_mode]
 
+    def _get_multiplexer_setup_options(  # pylint: disable=no-self-use # cause overrided in AEA
+        self,
+    ) -> Optional[Dict]:
+        """
+        Get dict of multiplexer setup options.
+
+        :return: dict of kwargs for Multipelxer.setup
+        """
+        return None
```

Comment:
we should have here:
```
return {"connections": self.active_connections}
```

Reply:
and then we can remove the pylint warning as well!

Reply:
nice point, requires small update on Multiplexer.setup to make default routing optional